peachjam/models/judgment.py

```diff
@@ -101,6 +102,20 @@ class Court(models.Model):
         "peachjam.EntityProfile", verbose_name=_("profile")
     )
     order = models.IntegerField(_("order"), null=True, blank=True)
+    country = models.ForeignKey(
+        Country,
+        null=True,
+        blank=True,
+        on_delete=models.CASCADE,
```

Comment:
Why cascade this, but protect the locality? I think better to protect against deleting both.